src/funcTest/groovy/info/solidsoft/gradle/pitest/functional/PitestPluginGradleVersionFunctionalSpec.groovy

```diff
@@ -25,84 +26,96 @@ import static info.solidsoft.gradle.pitest.PitestTaskConfigurationSpec.PIT_PARAM
+        PIT_PARAMETERS_NAMES_NOT_SET_BY_DEFAULT.each { parameterName ->
+            assert !result.standardOutput.contains("${parameterName}=")
+        }
+
         where:
-            requestedGradleVersion << applyJavaCompatibilityAdjustment(resolveRequestedGradleVersions()).unique()
+        requestedGradleVersion << applyJavaCompatibilityAdjustment(resolveRequestedGradleVersions()).unique()
     }
 
-    @IgnoreIf({new PreconditionContext().javaVersion >= 13 })   //There is no unsupported version of Gradle which can be used with Java 13
-    def "should fail with meaningful error message with too old Gradle version"() {
+    @IgnoreIf({ new PreconditionContext().javaVersion >= 13 })   //There is no unsupported version of Gradle which can be used with Java 13
+    void "should fail with meaningful error message with too old Gradle version"() {
         given:
-            gradleVersion = "5.5.1"
+        gradleVersion = '5.5.1'
+
         and:
-            assert PitestPlugin.MINIMAL_SUPPORTED_GRADLE_VERSION.compareTo(GradleVersion.version(gradleVersion)) > 0
+        assert PitestPlugin.MINIMAL_SUPPORTED_GRADLE_VERSION > GradleVersion.version(gradleVersion)
+
         when:
-            copyResources("testProjects/simple1", "")
+        copyResources('testProjects/simple1', '')
+
         then:
-            fileExists('build.gradle')
+        fileExists('build.gradle')
+
         when:
-            ExecutionResult result = runTasksWithFailure('tasks')
+        ExecutionResult result = runTasksWithFailure('tasks')
+
         then:
-            verifyAll {
-                def root = Exceptions.getRootCause(result.failure)
-                root.class.name == GradleException.name //name to mitigate differences on classloader
-                root.message.contains("'info.solidsoft.pitest' requires")
-                result.standardOutput.contains("WARNING. The 'info.solidsoft.pitest' plugin requires")
-            }
+        verifyAll {
+            Throwable root = Exceptions.getRootCause(result.failure)
+            root.class.name == GradleException.name //name to mitigate differences on classloader
+            root.message.contains("'info.solidsoft.pitest' requires")
+            result.standardOutput.contains("WARNING. The 'info.solidsoft.pitest' plugin requires")
+        }
     }
 
     //To prevent failure when Spock for Groovy 2.4 is run with Groovy 2.3 delivered with Gradle <2.8
     //Spock is not needed in this artificial project - just the test classpath leaks to Gradle instance started by Nebula
-    private static final Pattern SPOCK_JAR_PATTERN = Pattern.compile(".*spock-core-1\\..*.jar")
+    private static final Pattern SPOCK_JAR_PATTERN = Pattern.compile('.*spock-core-1\\..*.jar')
     private static final Predicate<URL> FILTER_SPOCK_JAR = { URL url ->
-        return !url.toExternalForm().matches(SPOCK_JAR_PATTERN)
+        !url.toExternalForm().matches(SPOCK_JAR_PATTERN)
```

Comment:
With most of the (short) lambdas, I'm fine to omit `return`.

Reply:
Since I'm disabling UnnecessaryReturnKeyword rule I'll revert the change for short lambdas too to keep the diff smaller while complying with the final CodeNarc config